frontend/src/components/Workspace/Visualize/VisualizeItemLayoutMenuIcon.tsx

```diff
@@ -10,13 +10,16 @@ import ListItemText from "@mui/material/ListItemText"
 import Menu from "@mui/material/Menu"
 import MenuItem from "@mui/material/MenuItem"
 
+import { cancelRoiApi } from "api/outputs/Outputs"
```

Comment:
@itutu-tienday I see this error seems to be caused by VisualizeItemSelectors.ts line 154
"throws new Error ("VisualaizeItemType is invalid")"
I think it will return null

Reply:
@quanpython 
Sorry, I may not have understood the intent of your question,
Is this error not a glitch, but rather a specification that needs to be sorted out first?

Also, I am assuming that this error did not occur before, but what are the factors that are causing this error now?

Reply:
@itutu-tienday 
I see there is a function to check the condition
export function isImageItem(item: VisualaizeItemType): item is ImageItem {
  return (
    item.itemType === VISUALIZE_ITEM_TYPE_SET.DISPLAY_DATA &&
    item.dataType === DATA_TYPE_SET.IMAGE
  )
}
But when selecting data, it returns dataType = "roi", so there is such an error. Are the above conditions correct? If so, I think there should be an adjustment on the server side.

Reply:
@ReiHashimoto 
すみません、上記の質問内容について、コメントが可能であれば お願いしたいと思います。
 （現在、ROI Edit のブランチで動作させると、FEでSystem errorが生じている状況の件です）

Reply:
@sanglevinh 
As you mentioned, `selectRoiItemFilePath` is expected to be called on ImagePlot.
In this file, multiple data types would be passed.
How about calling `cancelRoiApi` on ImagePlot.tsx? (using useEffect hooks)

Reply:
@ReiHashimoto Sorry, I still don't understand what you mean, about cancelRoiApi has been replaced with dispatch(cancelRoi(...))